shared/clients/email/helper.go

```diff
@@ -14,12 +14,13 @@ import (
 )
 
 const (
+	VERIFY_EMAIL_TEMPLATE_PATH   = "../shared/templates/verify_email.html"
 	QR_CODE_TEMPLATE_PATH        = "../shared/templates/email.html"
 	RESET_PASSWORD_TEMPLATE_PATH = "../shared/templates/forgot_password.html"
-	INVITE_TITLE                 = "Despedida Veras - Invite"
+	INVITE_TITLE                 = "Despedida Veras - Convite"
 	TICKETS_TILE                 = "Despedida Veras - Ingressos"
-	RESET_PASSOWRD_TITLE         = "Despedida Veras - Esqueci minha senha ?"
-	CONFIRM_EMAIL_TITLE          = "Despediva Veras - Confirme seu email"
+	RESET_PASSOWRD_TITLE         = "Despedida Veras - Esqueci minha senha"
+	VERIFY_EMAIL_TITLE           = "Despedida Veras - Confirme seu email"
```

Comment:
Take care on your next rebases, this line was added from another PR that doesn't have any relation with `verify email`